templates/searchadmin/search/quicksearch/info_table.html.twig

```diff
@@ -49,15 +49,17 @@
             {% endif %}
         {% endif %}
 
-        {% set owner = pimcore_user(element.getUserOwner()) %}
+        {% set userOwnerId = element.getUserOwner() %}
+        {% set owner = userOwnerId ? pimcore_user(userOwnerId) : null %}
```

Comment:
this looks like it would break the table column on the rows where the userOwner is missing, when `owner` is `null`

Reply:
What do you mean? If the owner is null the row is skipped. This was the behaviour also before Pimcore 11.

Reply:
ahh, you're right.
I've overlooked the `<tr>`, thought it was building a table with multiple columns